package.json

```diff
@@ -1,51 +1,68 @@
 {
-  "name": "ovh-angular-contact",
+  "name": "@ovh-ux/ng-ovh-contact",
```

Comment:
I think we can already replace all `ovh-angular-contact` by `ng-ovh-contact` (due to the renaming of this repository).

Is that make sense for you to do bit before merging it?

Reply:
Done ;)